source/kraken/worker.cpp

```diff
@@ -828,6 +828,94 @@ pbnavitia::Response Worker::car_co2_emission_on_crow_fly(const pbnavitia::CarCO2
     return r;
 }
 
+type::EntryPoint make_entry_point(const std::string& place,
```

Comment:
`create_entry_point` and `make_entry_point` have very similar names. Merge them or rename one to be explicit.